### DISCUSSION TITLE
CYBR 502 - Setting up Security Onion

### DISCUSSION THREAD

**Discussion by loopieshoes** [Security-Onion-Solutions/securityonion#9953] [Unsupported Versions]
I have downloaded and installed Security Onion. I have downloaded the following file. 

wget https://download.netresec.com/pcap/maccdc2012/maccdc2012_00000.pcap.gz --no-check-certificate

I have downloaded the file. 

I am then given instructions to UNZIP the pcap file. 

this is the instruction for that:

Unzip the PCAP file - Type: gzip -d maccdc2012_00000.pcap.gz

I then receive an error that there is an 
gzip: maccdc2012_00000.pcap.gz: unexpected end of file

I can login into Security Onion but the page I need is not setup yet because I have not completed this step. 

Can someone please help?

![download pcap to gzip pcap](https://user-images.githubusercontent.com/116060002/226046121-82603273-5363-470e-abd1-3840ad8e3e42.jpg)

**Comment by cm-ops** [comment]
You could try and `zcat` the file to see if there is an issue with it, if that works try `zcat file.gz > file`.
